src/main/java/io/github/thebusybiscuit/slimefun4/implementation/items/misc/StrangeNetherGoo.java

```diff
@@ -37,4 +44,18 @@ public int getBarteringLootChance() {
         return chance.getValue();
     }
 
+    @Nonnull
+    @Override
+    public ItemUseHandler getItemHandler() {
+        return this::use;
+    }
+
+    public void use(PlayerRightClickEvent e) {
```

Comment:
First off: This should NOT be public.
Also a missing nullability annotation.
But most importantly, there is no need for this method, this can all easily be done as a lambda expression.